php/tests/WellKnownTest.php

```diff
@@ -84,6 +84,7 @@ public function testAny()
         // Test is.
         $this->assertTrue($any->is(TestMessage::class));
         $this->assertFalse($any->is(Any::class));
+        $this->assertFalse($any->is(NotMessage::class));
```

Comment:
I think this should throw error, not just returning false here.

Reply:
Maybe also test a message not in the generated pool.

Reply:
I agree with @TeBoring, I think both the C extension and the PHP library should add an error message.

If we just return `false`, there is a risk that the function will return `false` when it should have returned `true` if a given message class has not been constructed yet.

I don't think this change will break anybody.